app/assets/javascripts/application.js

```diff
@@ -22,6 +22,7 @@
 //= require components
 //= require tablesort
 //= require tablesort/dist/sorts/tablesort.number.min
+//= require hide-teleconsultation-fields.js
```

Comment:
Can we drop the `js` extension? Also, could we call it just `teleconsultation-fields`, or `facility-teleconsultation-fields`, as it hides _and_ shows the fields?